src/magic.cpp

```diff
@@ -566,6 +571,9 @@ std::string spell::name() const
 
 std::string spell::message() const
 {
+    if ( alt_message.empty() == false ) {
```

Comment:
```suggestion
    if ( !alt_message.empty() ) {
```

Reply:
Is there a style rule that rejects the usage of == false? I am in favor of the long form because it is a lot clearer to read and also isn't a typo away from changing meaning

Reply:
keeping it for clarity